src/components/cart-card/CartCard.jsx

```diff
@@ -0,0 +1,34 @@
+import { useWishList } from 'context/WishListContext'
+import React from 'react'
+import './CartCard.css'
+
+export const CartCard = ({ _id,brand, title, discountedPrice, actualPrice, discountoff, srcimg, rating}) => {
+    const { addToWishList , wishlist, removeFromCart} = useWishList();
+
+  return (
+    <>
+        <div className="card-horizontal">
+
+            <div className="card-horizontal-container">
+                <div className="horizonal-image">
+                    <img src={ srcimg } alt="shoe image" className="card-horizontal-img" />
```

Comment:
**Suggestion:** Try to use underscore or dash  or camelCase in variable consisting of multiple word like here src_img or src-img or srcImg.